client/src/components/Menu/Menu.scss

```diff
@@ -0,0 +1,638 @@
+        margin-left: 30px;
+
+        @include media-breakpoint-down(sm) {
+          margin-left: 38px;
+        }
+      }
+
+      span.toggle-children {
+        top: 0;
+      }
+
+    }
+
+    &.current { //need to apply current stlye to flyout also (at least line height)
+
+      &>a {
+        background-color: $base-menu;
+
+        .toggle-children {
+          .toggle-children-icon {
+            @extend .icon-sprites-32x32;
+            @include sprite($sprites-32x32-menu-arrow-down);
+          }
+        }
+      }
+
+      a {
+        background-color: $base-menu;
+      }
+
+      ul {
+        border-top: 0;
+        display: block;
+      }
+
+      li {
+        background-color: $base-menu-bg;
+
+        a {
+          &.current,
+          &:hover {
+            background-color: $base-menu;
+          }
+        }
+
+      }
+    }
+
+    // Has sub menu
+    &.children {
+
+      .no-icon.text {
+        margin-left: 30px;
+      }
+
+      // Sub menu
+      ul li a {
+        font-size: 12px;
+        line-height: 18px;
+        height: 18px;
+        padding-top: 11px;
+        padding-bottom: 11px;
+        color: lighten($color-text-default, 10%);
+        background-color: $base-menu-bg;
+
+        span.text {
+          margin-left: 23px;
+        }
+      }
+
+      &.current {
+        > a {
+          background-color: transparent;
+
+          &:hover {
+            background-color: lighten($base-menu, 3%);
+          }
+        }
+      }
+
+      &.opened {
+        a:hover {
+          background-color: lighten($base-menu, 3%);
+        }
+
+        .current a {
+          background-color: $base-menu;
+        }
+
+        .opened .toggle-children-icon {
+          transform: rotate(45deg);
+        }
+      }
+    }
+
+    /* Style applied to the menu flyout only when the collapsed setting */
+    .collapsed-flyout {
+      left: 59px;
+      margin-top: -52px;
+      position: fixed;
+      width: 160px;
+      z-index: 1;
+      display: none;
+
+      .icon {
+        display: none;
+      }
+
+      li {
+        a {
+          background: $base-menu-bg;
+          box-shadow: none;
+          padding: 10px 0;
+          height: 40px;
+          line-height: 18px;
+
+          &:hover,
+          &:active {
+            background: lighten($base-menu, 3%);
+          }
+        }
+
+        &.children a {
+          font-size: 13px;
+          height: 18px;
+          cursor: default;
+        }
+
+        &.current {
+          a {
+            background-color: $base-menu;
+          }
+
+          &.children a {
+            background-color: $base-menu-bg;
+          }
+        }
+      }
+    }
+  }
+
+  &.collapsed {
+
+    .toggle-children {
+      display: none;
+    }
+
+    .opened {
+      ul.collapsed-flyout {
+        border-radius: 0 2px 2px 0 ;
+        border: 1px solid #d2d5d8;
+        box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+
+        li {
+          &.clone {
+            a {
```

Comment:
this could be `&.clone a {`
this gives a warning in our linter about too many levels of nesting